project/Dependencies.scala

```diff
@@ -101,8 +101,18 @@ object Dependencies {
   val capiAws = "com.gu" %% "content-api-client-aws" % "0.7.4"
 
   // Forcing a version of this to fix an issue with the dependency.
-  // This is a transitive dependency of the AWS SDK used by etag-caching library
-  val nettyCodecHttp2 = "io.netty" % "netty-codec-http2" % "4.1.112.Final"
+  // This is a transitive dependency of the play framework
+  val nettyVersion = "4.1.118.Final"
+  val netty = Seq(
+    "io.netty" % "netty-handler" % nettyVersion,
+    "io.netty" % "netty-codec-http" % nettyVersion,
+    "io.netty" % "netty-buffer" % nettyVersion,
+    "io.netty" % "netty-common" % nettyVersion,
+    "io.netty" % "netty-transport-classes-epoll" % nettyVersion,
+    "io.netty" % "netty-transport-native-unix-common" % nettyVersion,
+    "io.netty" % "netty-transport" % nettyVersion,
+    "io.netty" % "netty-transport-native-epoll" % nettyVersion,
+  )
```

Comment:
You can avoid some repetition:

```suggestion
  val netty = Seq(
    "netty-handler",
    "netty-codec-http",
    "netty-buffer",
    "netty-common",
    "netty-transport-classes-epoll",
    "netty-transport-native-unix-common",
    "netty-transport",
    "netty-transport-native-epoll",
  ).map("io.netty" % _ % "4.1.118.Final")
```